dcos/auth.py

```diff
@@ -58,10 +57,16 @@ def _prompt_user_for_token(url, token_type):
     :rtype: str
     """
 
-    msg = "\n{}\n\n    {}\n\nEnter {}:".format(
-          "Please go to the following link in your browser:",
+    msg = "\n{}\n\n{}\n\n    {}\n\nEnter {}:".format(
+          "Attempting to automatically open a page in your browser...",
+          "If it doesn't work, open the following link:",
```

Comment:
How about:
`If you were not redirected to your browser, please go to the following link: `